eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxWits.hs

```diff
@@ -153,6 +155,29 @@ instance EncCBORGroup RdmrPtr where
 instance DecCBORGroup RdmrPtr where
   decCBORGroup = RdmrPtr <$> decCBOR <*> decCBOR
 
+instance ToJSON RdmrPtr where
+  toJSON = \case
+    RdmrPtr Spend n ->
+      object
+        [ "kind" .= String "RedeemerPointerSpending"
+        , "value" .= n
+        ]
+    RdmrPtr Mint n ->
+      object
+        [ "kind" .= String "RedeemerPointerMinting"
+        , "value" .= n
+        ]
+    RdmrPtr Cert n ->
+      object
+        [ "kind" .= String "RedeemerPointerCertifying"
+        , "value" .= n
+        ]
+    RdmrPtr Rewrd n ->
+      object
```

Comment:
Same observation about `kindObject` as before

Reply:
That was the reason why I added the function `kindObject`. That and also the fact that I like the name :smile: